03_letters/mohitnag/main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprintln(out, strings.Join(sortLetters(letters("aba")), "\n"))
+}
+
+func letters(s string) map[rune]int {
+	m := map[rune]int{}
+	rs := []rune(s)
```

Comment:
This step is not required. In go when you iterate over a string like `s`, it will treat each character in the string as a rune. Therefore you can directly assign to the `map[rune]int{}` from `s`. Can you please update this?

```go
func letters(s string) map[rune]int {
	m := map[rune]int{}
	for _, val := range s {
		// val here is a rune.
	}
	return m
}
```

Reply:
Done